### PR TITLE
Adding Web App Ssl binding cmdlets and renaming existing Web App cmdlet names to use 'Rm' instead of 'RM'

### PR DESCRIPTION
**Pull request by akurmi**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@akurmi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (akurmi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj [hunk 1]
```diff
@@ -179,6 +183,13 @@
     <None Include="SessionRecords\Microsoft.Azure.Commands.Websites.Test.ScenarioTests.AppServicePlanTests\TestSetAppServicePlan.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.Websites.Test.ScenarioTests.SSLBindingTests\TestCreateNewWebAppSSLBinding.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.Websites.Test.ScenarioTests.SSLBindingTests\TestGetNewWebAppSSLBinding.json" />
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj]: @akurmi These new recording files need to be copied to the output directory as well - in VS Properties/Copy If Newer

Comment (akurmi) [inline src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj]: Thanks Mark for identifying the issue. I will fix it and start a new PR.

Comment (akurmi) [inline src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj]: Thanks @markcowl  for identifying the issue, I just checked-in a fix.

**Comment by markcowl** [comment]
@akurmi  Merging. Please add in details of the changes to the changelog in a separate PR.

**Pull request merged**